backend/modules/query_controllers/example/payload.py

```diff
@@ -27,7 +26,6 @@
     "query": "Explain in detail different categories of credit cards",
     "model_configuration": {
         "name": "openai-main/gpt-3-5-turbo",
-        "provider": "truefoundry",
         "parameters": {"temperature": 0.1},
```

Comment:
What do think on changing this payload similar to ChatOpenAI?
```
model_config: {
- model_name
- provider
- temperature
- max_tokens
- model_kwargs
}
```

temperature and max_tokens can go within parameters if required?